### PR TITLE
Submit Double to Valora for Dapp listing

### PR DESCRIPTION
**Pull request by h3he**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [X] connects to Valora via WalletConnect V2;
- [X] must be mobile-first or mobile responsive;
- [X] is in one of the existing categories; and
- [X] includes a link to user documentation.

For more information see the [Valora Listing Overview](https://docs.valora.xyz/listing/overview).

### CODE REVIEW

**Comment by MuckT** [comment]
@h3he I am unable to find an link to user documentation on your site.

**Comment by h3he** [comment]
> @h3he I am unable to find an link to user documentation on your site.

Here is the user guide: https://www.double2win.xyz/docs/. There is no place for me to put that in the PR though.

**Comment by MuckT** [comment]
@h3he the docs look good, but they should be accessible on https://app.double2win.xyz.

**Comment by h3he** [comment]
> @h3he the docs look good, but they should be accessible on https://app.double2win.xyz.

hmm, that would break our beautiful design. Do we have to have user guide access from the app interface?

**Comment by h3he** [comment]
I think the Mento v2 (which is listed on Valora) does not have link to doc either: https://app.mento.org/

**Comment by MuckT** [comment]
@h3he Mento has links to their Discord, Twitter and GitHub in the bottom left corner of the site.
<p align="center">

</p>

**Comment by h3he** [comment]
> @h3he Mento has links to their Discord, Twitter and GitHub in the bottom left corner of the site.
> 
> > Your Dapp:
> > ...includes a link to user documentation.
> 
> 

Can we follow the same approach used by Mento? We can make the logo clickable and redirect to our website www.double2win.xyz and then it has link to double2win.xyz/docs or redirect directly to double2win.xyz/docs? Will this change satisfy the requirement?

**Comment by MuckT** [comment]
@h3he a link to your homepage, [www.double2win.xyz](http://www.double2win.xyz/) - which contains a link to user documentation, in the footer of  https://app.double2win.xyz for me.

**Comment by h3he** [comment]
> @h3he a link to your homepage, [www.double2win.xyz](http://www.double2win.xyz/) - which contains a link to user documentation, in the footer of https://app.double2win.xyz for me.

I am not 100% clear here. So I like to confirm. Can we link to our homepage [www.double2win.xyz](http://www.double2win.xyz/) via the Double logo on the top left of the https://app.double2win.xyz/?

**Comment by MuckT** [comment]
@h3he linking to http://www.double2win.xyz from the double icon seems fine for now. While it's not as clear that it's a link compared to the icons seen on Mento it does provide a way for the users to access the documentation. If we get complaints from users we might re-address.

**Comment by h3he** [comment]
> @h3he linking to http://www.double2win.xyz from the double icon seems fine for now. While it's not as clear that it's a link compared to the icons seen on Mento it does provide a way for the users to access the documentation. If we get complaints from users we might re-address.

Sounds good. The Dapp front-end has been updated. Can you verify? If users complain, we will adjust our UI design to address users' complaints. Thank you!

**Comment by MuckT** [comment]
@h3he it looks like the double icon is still redirecting to https://app.double2win instead of http://www.double2win.xyz.

**Comment by h3he** [comment]
> @h3he it looks like the double icon is still redirecting to https://app.double2win.xyz instead of http://www.double2win.xyz.

Please clean the cache and try it again? I have verified myself.

File: locales/base.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "resource": "Get rewards by supporting small businesses",
     "ethichub": "Lend capital to unbanked farmers through DeFi crowdfunding",
     "uniswap": "Swap and get rewards by providing liquidity",
+    "double": "Get rewards and earn yields from unused tokens by providing liquidity",
```
Comment (ellenwoconnor) [inline locales/base.json:40]: Legal does not allow us to talk about earning yield, can we keep this as "get rewards by providing liquidity"? (wasn't sure what "unused tokens" meant so I omitted it)

Comment (h3he) [reply locales/base.json:40]: Sounds good.

Comment (MuckT) [reply locales/base.json:40]: @h3he you'll need to make this change as I don't have write access to your fork.

Comment (h3he) [reply locales/base.json]: OK. Just made the change. Please review and confirm.

**Comment by MuckT** [comment]
@h3he. I noticed after connecting or withdrawing the dapp does not appear connected; however, after I refresh the page my it correctly shows as connected. Can you take a look at this?

**Comment by h3he** [comment]
> @h3he I noticed after connecting or withdrawing the dapp does not appear connected; however, after I refresh the page it correctly shows as connected. Can you take a look at this?

We noticed this issue as well. And Geo pinged me last week on the same issue as well. We investigated and concluded this was not anything related to Double product. It is related to WalletConnect v2, which could not provide a stable connection/service. Geo will make a note to Valora users to refresh the page and try it again. Unfortunately, this is not the ideal UX that we all like to see and have.

**Comment by MuckT** [comment]
@h3he other dapps are using WalletConnect v2 and do not have this issue. If you're using `@web3modal/ethereum` you can act on state changes of the account with something similar to the below code:

```TypeScript
import { EthereumClient } from '@web3modal/ethereum'

ethereumClient.watchAccount(async ({ address, connector, status }) => {
// Action on client changes here e.g.
  if(status === 'connected') {
  // Do some stateful things for connected account.
  }
})
```

**Comment by h3he** [comment]
Sorry I might misunderstand your comment before. It seems like a different issue that the Valora needs to be reconnected via WalletConnect again. The issue Geo and we encountered was "Valora has trouble even connecting to Double and need to try it again". We will look into it.

**Comment by h3he** [comment]
@MuckT I just tested myself. After withdraw using Valora wallet, the Valora wallet is still connected for me. Would it possible for you to capture the video of the issue?

**Comment by MuckT** [comment]
@h3he, thanks for taking another look! Here's a video of the connection flow - the behavior I'd expect is that after connecting the dapp would show as connected without the need to tap connect > wallet connect again. The behavior after a submitting an action through the dapp is similar where I need to refresh the page to see an updated balance.

https://github.com/valora-inc/dapp-list/assets/26950305/2e9de418-42b5-4637-824e-c77eb50811e0

**Comment by h3he** [comment]
Thanks. The video is very helpful.

**Comment by h3he** [comment]
@MuckT We fixed the issue and made a new release. Can you give another try? Please do hard fresh of the mobile browser to clean all cache. Thanks.

**Comment by MuckT** [comment]
@h3he thanks for taking the time to fix the issues reported! 🚀

**Review by MuckT — APPROVED** [review]

**Pull request merged**